client/src/Header.jsx

```diff
@@ -23,10 +28,30 @@ class Header extends React.Component {
     return (
       <div>
         <div className="title">WhaleThen</div>
-        <br/>
+        <br />
         <div className="container timelineParams">
           <div className="label">{timelineName}</div>
           <div className="label">{timelineId}</div>
+          <MuiThemeProvider>
+            <CopyToClipboard
+              onCopy={this.onCopy}
+              text={`http://localhost:3002/#/${timelineId}`}
```

Comment:
Currently hardcoded - will need to change right after re-deploying to our deployed url.